crates/biome_js_analyze/tests/specs/nursery/noSecrets/invalid.js.snap

```diff
@@ -0,0 +1,219 @@
+---
+source: crates/biome_js_analyze/tests/spec_tests.rs
+expression: invalid.js
+---
+# Input
+```jsx
+const awsApiKey = "AKIA1234567890EXAMPLE"
+const slackToken = "xoxb-123456789012-123456789012-abcdefghijklmnopqrstuvwx";
+const rsaPrivateKey = "-----BEGIN RSA PRIVATE KEY-----\nMIIEpAIBAAKCAQEA1234567890..."
+const facebookToken = "facebook_app_id_12345abcde67890fghij12345";
+const twitterApiKey = "twitter_api_key_1234567890abcdefghijklmnopqrstuvwxyz";
+const githubToken = "github_pat_1234567890abcdefghijklmnopqrstuvwxyz";
+const clientSecret = "abcdefghijklmnopqrstuvwxyz"
+const herokuApiKey = "heroku_api_key_1234abcd-1234-1234-1234-1234abcd5678";
+const genericSecret = "secret_1234567890abcdefghijklmnopqrstuvwxyz";
+const genericApiKey = "api_key_1234567890abcdefghijklmnopqrstuvwxyz";
+const slackKey = "https://hooks.slack.com/services/T12345678/B12345678/abcdefghijklmnopqrstuvwx"
+const twilioApiKey = "SK1234567890abcdefghijklmnopqrstuv";
+const dbUrl = "postgres://user:password123@example.com:5432/dbname";
+```
+
+# Diagnostics
+```
+invalid.js:1:19 lint/nursery/noSecrets ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! Potential secret found.
+  
+  > 1 │ const awsApiKey = "AKIA1234567890EXAMPLE"
+      │                   ^^^^^^^^^^^^^^^^^^^^^^^
+    2 │ const slackToken = "xoxb-123456789012-123456789012-abcdefghijklmnopqrstuvwx";
+    3 │ const rsaPrivateKey = "-----BEGIN RSA PRIVATE KEY-----\nMIIEpAIBAAKCAQEA1234567890..."
+  
+  i AWS API Key
```

Comment:
You should take a look at the tips we have for writing good lint messages: https://github.com/biomejs/biome/blob/main/crates/biome_analyze/CONTRIBUTING.md#explain-a-rule-to-the-user

Reply:
Updated!